src/ng/controller.js

```diff
@@ -142,8 +144,7 @@ function $ControllerProvider() {
           addIdentifier(locals, identifier, instance, constructor || expression.name);
         }
 
-        var instantiate;
-        return instantiate = extend(function $controllerInit() {
+        var instantiate = extend(function $controllerInit() {
```

Comment:
We don't need to even need the extra var. We could as well return directly without assigning.